pages/glossary/essential-text-change.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Essential Text Change
+key: essential-text-change
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+An _essential text change_ occurs when the text content of an HTML element is required to convey accurate information to the user. An _essential text change_ may appear in any of the following forms:
```

Comment:
I find this to be awkward, but I also couldn't figure out a good way to say it without doing something like:
> An essential text change occurs when the changing of text on a page is essential.